rel/files/stanchion.schema

```diff
@@ -322,6 +322,32 @@
   hidden
 ]}.
 
+%% == Supercluster aka Multi-bug Support ==
+
+%% @doc IP and port for supercluster members.  Specify one member for
+%% each line, in which the last token of key is member identifier and
+%% value is a PB address of Riak node to connect.
+{mapping, "supercluster.member.$member_id", "stanchion.bags", [
+  {datatype, ip},
+  {include_default, "bag-A"},
```

Comment:
The best way is to match the string here with one in the doc (in the future.) This is *not user-facing* because of `hiddden`, how about leaving as it is and defer change ?